documentation/docs/documentation/readme.md

```diff
@@ -1,7 +1,45 @@
 # Installation
 
-```bash
+You can install biome-text with pip or by installing from source.
+
+
+## Pip
+
+You can install everything required for library. This is often the right choice:
+```shell
 pip install biome-text
 ```
 
+## Install from Source
+To install biome-text from source, clone the repository from github:
+
+````shell
+git clone https://github.com/recognai/biome-text.git
+cd biome-text
+python -m pip install .
+````
+
+If `make` command is enabled in your system, you can use already defined make directives:
```

Comment:
If the `make` command is enabled in your system, you can use the following make directives: